src/NewsletterEpic/index.tsx

```diff
@@ -12,9 +12,71 @@ import {
 } from '@guardian/source-foundations';
 import { COMPONENT_NAME, canRender } from './canRender';
 import type { TrackClick } from '../utils/tracking';
-import { CTA, NewsletterFrequency } from '../newsletterCommon';
+import { NewsletterFrequencyBlock } from '../components/NewsletterFrequencyBlock';
+import { NewsletterCtaButton } from '../components/NewsletterCtaButton';
 import { NewsletterSubscribeCallback } from '../types/dcrTypes';
 
+export { COMPONENT_NAME };
+
+export type BrazeMessageProps = {
+    header?: string;
+    frequency?: string;
+    paragraph1?: string;
+    paragraph2?: string;
+    imageUrl?: string;
+    newsletterId?: string;
+    ophanComponentId?: string;
+};
+
+export type Props = {
+    brazeMessageProps: BrazeMessageProps;
+    subscribeToNewsletter: NewsletterSubscribeCallback;
+    trackClick: TrackClick;
+};
+
+export const NewsletterEpic: React.FC<Props> = (props: Props) => {
+    const {
+        brazeMessageProps: {
+            header,
+            frequency,
+            paragraph1,
+            paragraph2,
+            imageUrl,
+            newsletterId,
+            ophanComponentId,
+        },
+        subscribeToNewsletter,
+        trackClick,
+    } = props;
+
+    if (!canRender(props.brazeMessageProps)) {
+        return null;
+    }
+
+    return (
+        <ThemeProvider theme={brand}>
+            <section css={styles.epicContainer}>
+                <div>
+                    <img css={styles.image} src={imageUrl}></img>
+                </div>
+                <div css={styles.rightSection}>
+                    <span css={styles.heading}>{header}</span>
+                    <NewsletterFrequencyBlock frequency={frequency} />
+                    <p css={styles.paragraph}>{paragraph1}</p>
+                    {paragraph2 ? <p css={styles.paragraph}>{paragraph2}</p> : null}
+                    <NewsletterCtaButton
+                        subscribeToNewsletter={subscribeToNewsletter}
+                        newsletterId={newsletterId as string}
+                        ophanComponentId={ophanComponentId}
+                        trackClick={trackClick}
+                        newsletterCta="Sign up"
+                    />
+                </div>
+            </section>
+        </ThemeProvider>
+    );
+};
+
```

Comment:
Git isn't doing a good job of showing the differences here - is this just a move of a large chunk of code or are there changes in here?

Reply:
The only thing I did beyond importing and using the components (with their updated names) was move the styling to the bottom of the file ... Bad GitHub!